src/modules/launcher/Plugins/Microsoft.PowerToys.Run.Plugin.TimeDate/Components/ResultHelper.cs

```diff
@@ -0,0 +1,92 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Globalization;
+using System.Runtime.CompilerServices;
+using System.Windows;
+using Microsoft.PowerToys.Run.Plugin.TimeDate.Properties;
+using Wox.Plugin;
+using Wox.Plugin.Logger;
+
+[assembly: InternalsVisibleTo("Microsoft.PowerToys.Run.Plugin.TimeDate.UnitTests")]
+
+namespace Microsoft.PowerToys.Run.Plugin.TimeDate.Components
+{
+    internal static class ResultHelper
+    {
+        /// <summary>
+        /// Get the string based on the requested type
+        /// </summary>
+        /// <param name="isSystemTimeDate">Does the user search for system date/time?</param>
+        /// <param name="stringId">Id of the string. (Example: `MyString` for `MyString` and `MyStringNow`)</param>
+        /// <param name="stringIdNow">Optional string id for now case</param>
+        /// <returns>The string from the resource file, or <see cref="string.Empty"/> otherwise.</returns>
+        internal static string SelectStringFromResources(bool isSystemTimeDate, string stringId, string stringIdNow = default)
+        {
+            if (!isSystemTimeDate)
+            {
+                return Resources.ResourceManager.GetString(stringId, CultureInfo.CurrentCulture) ?? string.Empty;
```

Comment:
Do we need CurrentUICulture here instead of CurrentCulture?

Reply:
Let's check what we're doing on the other calls to this API through PowerToys and be consistent.

Reply:
I think we should compare with system plugin and settings plugin. I am asking because @CleanCodeDeveloper uses UICulture in #16942. I have checked System plugin now and it uses UICulture too.

Reply:
I will change it later.

Reply:
this is fixed with commit faf578e

Reply:
Mee too. 😁

I realized this difference when seeing your pr. ythen I understand which culture the "region" setting is. 😅